distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Duration.enso

```diff
@@ -128,9 +154,11 @@ type Duration
 
              import Standard.Base.Data.Time.Duration
 
-             example_eq = (Duration.new seconds=60).total_minutes == (Duration.new minutes=1).total_minutes
-    == : Duration -> Boolean
-    == self that = self.equals_builtin that
+             example_eq = (Duration.seconds 60).total_minutes == (Duration.minutes 1).total_minutes
+    == : Duration -> Boolean ! Time_Error
+    == self that =
+        ensure_duration that <|
```

Comment:
On passing a non-duration I think we should just return false.

Reply:
Sounds good. No error throwing for non-duration then.